cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,63 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
+  const testData = {
+    user: {
+      username: 'YuliiaPyrih',
+      password: 'Mate1234!',
+    },
+    book: 'Speaking JavaScript',
+  };
+
   before(() => {
```

Comment:
Pay attention that you visit login page before each test that's why your 2,3 tests are failed. Please review

Reply:
Fixed